interface/patient_file/encounter/find_code_dynamic.php

```diff
@@ -0,0 +1,311 @@
+<?php
+/* Copyright (C) 2015-2017 Rod Roark <rod@sunsetsystems.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ */
+
+// For DataTables documentation see: http://legacy.datatables.net/
+
+$fake_register_globals = false;
+$sanitize_all_escapes  = true;
+
+require_once('../../globals.php');
+require_once($GLOBALS['srcdir'] . '/patient.inc');
+require_once($GLOBALS['srcdir'] . '/csv_like_join.php');
+require_once($GLOBALS['fileroot'] . '/custom/code_types.inc.php');
+
+$info_msg = "";
+
+// What we are picking from: codes, fields, lists or groups
+$what = empty($_GET['what']) ? 'codes' : $_GET['what'];
+
+// For what == codes
+$codetype = empty($_GET['codetype']) ? '' : $_GET['codetype'];
+if (!empty($codetype)) $allowed_codes = split_csv_line($codetype);
+// This is the html element of the target script where the selected code will be stored.
+$target_element = empty($_GET['target_element']) ? '' : $_GET['target_element'];
+
+// For what == fields
+$source = empty($_GET['source']) ? 'D' : $_GET['source'];
+
+// For what == groups
+$layout_id = empty($_GET['layout_id']) ? '' : $_GET['layout_id'];
+?>
+<html>
+<head>
+<?php html_header_show(); ?>
+<title><?php echo xlt('Code Finder'); ?></title>
+<link rel="stylesheet" href='<?php echo attr($css_header) ?>' type='text/css'>
+
+<style type="text/css">
+</style>
+
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/datatables.net-dt-1-10-13/css/jquery.dataTables.min.css" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/datatables.net-colreorder-dt-1-3-2/css/colReorder.dataTables.min.css" type="text/css">
+
+<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-10-2/index.js"></script>
```

Comment:
Is there are reason your using 1-10-2 jquery? much better if use 3-1-1 .

Reply:
Nope, no reason. Thanks.